src/NuGetForUnity/Editor/NugetConfigFile.cs

```diff
@@ -147,6 +178,11 @@ public void Save(string filePath)
             addElement.Add(new XAttribute("value", savedRepositoryPath));
             config.Add(addElement);
 
+            addElement = new XElement("add");
+            addElement.Add(new XAttribute("key", "PackagesConfigPath"));
+            addElement.Add(new XAttribute("value", RelativePackagesConfigPath));
```

Comment:
Should we really store the path even if it is just the default path?

Reply:
I think it would do no harm... It's changed to `.` now so it indicates the folder where `NuGet.config` is.